build.proj

```diff
@@ -5,7 +5,7 @@
     <IsTraversalBuild>true</IsTraversalBuild>
   </PropertyGroup>
   
-  <Import Project="dir.props" />
+  <Import Project="src\dir.props" />
```

Comment:
The samples are included in the .sln under src, even though they are in a different directory.  So they have the same solution as the rest of the projects.

I do think we should consider moving the samples folder under `src` (as a separate task).  Right now we're duplicating things between the dir.targets under `src/` and `Samples/`.